src/NServiceBus.Persistence.CosmosDB/NServiceBus.Persistence.CosmosDB.csproj

```diff
@@ -7,10 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Particular.CodeRules" Version="0.7.0" PrivateAssets="All" />
     <PackageReference Include="Particular.Packaging" Version="0.9.0" PrivateAssets="All" />
     <PackageReference Include="NServiceBus" Version="[7.1.10, 8.0.0)" />
-    <PackageReference Include="Microsoft.Azure.Cosmos" Version="[3.14.0, 4)" />
+    <PackageReference Include="Microsoft.Azure.Cosmos" Version="[3.15.0, 4)" />
```

Comment:
Why was this update included in this PR?

Reply:
Microsoft delisted 3.14.0 and as there was an issue. Trying to compile using 3.14.0 causes an error.